data/remnant/remnant 2 cognizance.txt

```diff
@@ -1183,6 +1183,8 @@ mission "Remnant: Cognizance 25"
 		dialog "You have destroyed the ships that the Palavret left behind."
 	on waypoint
 		dialog "You have patrolled all six systems."
```

Comment:
```suggestion
		dialog "You have patrolled all six systems. Time to return to <planet>."
```

Reply:
There are two requirements to the mission. If you just patrol with the out killing the chasers, it won’t mark as complete. That was the entire point of this pill request.